include/z64scene.h

```diff
@@ -74,87 +74,92 @@ typedef struct {
 typedef struct {
     /* 0x00 */ Gfx* opa;
     /* 0x04 */ Gfx* xlu;
-} MeshHeader01Entry; // size = 0x08
+} MeshHeaderDListsEntry; // size = 0x08
```

Comment:
`MeshHeaderDListsEntry` is used for both `MeshHeaderNormal` and `MeshHeaderPrerenderBase`

I kept "header" because I personally don't like "mesh" on its own, it sounds super generic, maybe we could replace all "meshHeader" occurrences with "roomMesh" though?

Reply:
Ive mentioned it in the past but I dont like the term mesh at all for this, personally. imo its too ambiguous and doesnt sound at all to me like the dlists for a room. mesh could very well also be referring to a collision mesh

Reply:
(changed "mesh header" to "room shape" but the comment on `RoomShapeDlistsEntry` having two uses still stands)